### ISSUE TITLE
PEP8 plugin Installation instructions for other editors

### ISSUE THREAD

**Issue by encukou** [pyvec/naucse.python.cz#311]
In https://github.com/pyvec/naucse.python.cz/pull/306, Lumír added initial instructions for installation of pep8 plugins for editors.

But the situation is not that simple. For Atom we now have simple step by step description and for other editors just a note about PEP8. But installation for Notepad++, Gedit and Kate requires a manual download of plugins and/or copying a script from various sources to right folders which may not exist on the filesystem etc.

* [ ] Notepad++ - https://kmonsoor.wordpress.com/2014/05/02/pep8_tonizer-on-for-notepad-make-your-python-code-pep8-compliant-on-editor/
* [ ] Gedit - https://github.com/khertan/gedit_flake8
* [ ] Kate - https://github.com/goinnn/Kate-plugins#id15

Question is whether it's a good idea to add the complicated guide for plugins installation to editors' pages when mentioned editors aren't used widely.

**Comment by hroncok** [comment]
I'd just add the links and a note that a coach will help you,  or contact
us if you need help and you study without a coach.

Dne 6. 2. 2018 11:21 dop. napsal uživatel "Petr Viktorin" <
notifications@github.com>:

In #306 <https://github.com/pyvec/naucse.python.cz/pull/306>, Lumír added
initial instructions for installation of pep8 plugins for editors.

But the situation is not that simple. For Atom we now have simple step by
step description and for other editors just a note about PEP8. But
installation for Notepad++, Gedit and Kate requires a manual download of
plugins and/or copying a script from various sources to right folders which
may not exist on the filesystem etc.

   - Notepad++ - https://kmonsoor.wordpress.com/2014/05/02/pep8_tonizer-
   on-for-notepad-make-your-python-code-pep8-compliant-on-editor/
   <https://kmonsoor.wordpress.com/2014/05/02/pep8_tonizer-on-for-notepad-make-your-python-code-pep8-compliant-on-editor/>
   - Gedit - https://github.com/khertan/gedit_flake8
   - Kate - https://github.com/goinnn/Kate-plugins#id15

Question is whether it's a good idea to add the complicated guide for
plugins installation to editors' pages when mentioned editors aren't used
widely.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/pyvec/naucse.python.cz/issues/311>, or mute the thread

**Comment by frenzymadness** [comment]
I agree with Miro. I don't want to spend time on editors which basically no one use. Also, complicated guides may change quite often and we'd have to maintain them as well.
